pyccl/baryons/baccoemu_baryons.py

```diff
@@ -0,0 +1,136 @@
+__all__ = ("BaccoemuBaryons",)
+
+import numpy as np
+
+from .. import Pk2D
+from . import Baryons
+
+
+class BaccoemuBaryons(Baryons):
+    """See https://arxiv.org/abs/2011.15018 and https://bacco.dipc.org/emulator.html
+    """
+    name = 'BaccoemuBaryons'
+    __repr_attrs__ = __eq_attrs__ = ("log10_M_c", "log10_eta", "log10_beta", "log10_M1_z0_cen", 
+                                     "log10_theta_out", "log10_theta_inn", "log10_M_inn")
+    def __init__(self, log10_M_c=14, log10_eta=-0.3, log10_beta=-0.22, log10_M1_z0_cen=10.5, 
+                 log10_theta_out=0.25, log10_theta_inn=-0.86, log10_M_inn=13.4):
+        import warnings
+        with warnings.catch_warnings():
```

Comment:
Is this to avoid triggering all the tensorflow warnings? And do all the lines below need to be within this context, or just the `import` statement?

Reply:
Yes, it's because of that. In fact only the import and initialisation need to be in this scope, the assignments (kmin, kmax, amin, amax) can be outside, so I moved them